README.adoc

```diff
@@ -150,10 +147,18 @@ only supported orchestrators are reported.
   * `stop`: Stop the given service in the box.
   * `restart`: Restart the given service in the box.
 +
---
 The supported options for `service` are: Docker, OpenShift.
 
+1. `vagrant service-manager install-cli [service] [--cli-version] [--path]`
++
+Install the client binary for the specified `service`.
++
+The possible options are:
 
+  * `--cli-version`: Download the specified binary version.
```

Comment:
If the download url is inaccessible then following error message will be printed.
```
Download URL is not accessible.
Possible reason: Invalid version name
```
As, there is no simple way to validate the valid versions. One hard way is to somehow get all the released versions list and validate against that list.